financialaid/api_test.py

```diff
@@ -57,6 +57,29 @@ def create_program():
     return program
 
 
+def create_tier_programs(program):
```

Comment:
this function is always called immediately after `financialaid.api_test.create_program`. that makes perfect sense bc it's another necessary step for creating a valid program. i think you should either get rid of this method and move the code inside `create_program`, or just invoke this method at the end of `create_program`